### ISSUE TITLE
Recategorise CVSS Score with Environmental Score

### ISSUE THREAD

**Issue by Spawney** [dependency-check/DependencyCheck#6730]
Is it possible to recategorise a CVSS score based on Environmental factors in some form of suppression type file?

**Comment by aikebah** [comment]
No, CVSS scores reported by this tool are always the static base scores, there is no support for reporting an environment-adjusted score.
